### DISCUSSION TITLE
Dumb question alert. Entities and their desciptions

### DISCUSSION THREAD

**Discussion by dougle03** [BottlecapDave/HomeAssistant-OctopusEnergy#703] [Q&A]
Is there a list somewhere of all the possible entities the integration creates and a detailed explanation of what each entity actually means?
Sorry if I've missed that somewhere, but with such long entity names the standard integration screen is a little overwhelming.
Particularly, I have an entity in my gas list called 'previous accumulative **cost override**' that is significantly larger than the 'previous **accumulative cost**' entity. I don't understand why one is higher than the other....

Thanks

**Comment by BottlecapDave** [comment]
Hi there and welcome. Theres a large amount of [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/) available, including the available entities. These are broken down into areas, for example [electricity](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/)
